models/aeif_cond_beta_multisynapse.h

```diff
@@ -208,14 +209,62 @@ class aeif_cond_beta_multisynapse : public Archiving_Node
   void get_status( DictionaryDatum& ) const;
   void set_status( const DictionaryDatum& );
 
+  static inline Name
```

Comment:
Do not inline, make private and move implementation to cpp-file.

Reply:
Done in eb41277